docs/GDD/OPENCOLD.md

```diff
@@ -84,9 +84,9 @@ Revision: 0.0.1
     - Total Costs with breakdown
 
 ## Influences (Brief)
-  - Influence #1
-    - Medium (Television, Games, Literature, Movies, etc.)
-    - /Explain why this is an influence in 1 Paragraph or less/
+  - Zero Sievert 
+    - Game
+    - Resource management system,
   - Influence #2
```

Comment:
Additional influence is definitely [Outer Wilds](https://store.steampowered.com/app/753640/Outer_Wilds/) given the "you have X time until the world dies and everything* restarts" USP.